editor/animation/animation_blend_space_2d_editor.cpp

```diff
@@ -353,10 +413,14 @@ void AnimationNodeBlendSpace2DEditor::_add_menu_type(int p_index) {
 		return;
 	}
 
+	// Trim 'AnimationNode' from class name.
+	String unique_name = _generate_unique_blend_point_name(blend_space, node->get_class().substr(13));
```

Comment:
```suggestion
	String unique_name = _generate_unique_blend_point_name(blend_space, node->get_class().replace_first("AnimationNode", "");
```
To be clear and for consistency with StateMachineEditor.